### PR TITLE
pbr: update to 1.1.6-20

### PR DESCRIPTION
**Pull request by stangri**
Maintainer: me
Compile tested: x86_64, Dell EMC Edge620, OpenWrt 23.05.4
Run tested: x86_64, Dell EMC Edge620, OpenWrt 23.05.4

Description:
This version is the final version supporting iptables and:
* it separates the old iptables/nft-capable init script from the new nft-only init script
* the new nft-script is a significant rewrite of the old recursive calls/policy parsing and tries to create inlline nft sets which offers performance improvements

### CODE REVIEW

File: net/pbr/Makefile [hunk 1]
```diff
@@ -32,7 +31,6 @@ $(call Package/pbr/default)
   DEPENDS+=+kmod-nft-core +kmod-nft-nat +nftables-json
   VARIANT:=nftables
   DEFAULT_VARIANT:=1
-  PROVIDES+=vpnbypass vpn-policy-routing
```
Comment (BKPepe) [inline net/pbr/Makefile:35]: Is this change intentional? If yes, it was not described in the commit description together with the CONFLICTS removal.

Comment (stangri) [reply net/pbr/Makefile:35]: It is. This update includes 4000-ish new lines and 1000-ish removed lines from upstream repo. I'd appreciate it if you went line by line and elaborated on what is expected to be in the commit/PR description and what isn't.

Comment (BKPepe) [reply net/pbr/Makefile:35]: So, you are asking that I or rather we should visit your upstream repository and review every single commit, which you do. I dont think this is how it works. 

No, your change it is not intented. I do have struggle that you are not used to using provides and conflicts, otherwise you knows how to use it. You removed your old deprecated packages, then introduced new ones, basically simillar ones as they were refactored. So, if users are using automatic updates (its case of Turris routers) or different kind of updates, they will lose your package and I dont think they will browse to its source code to know whats happen.

Comment (BKPepe) [reply net/pbr/Makefile:35]: Contributing isnt a new thing, no need to reinvent a wheel. Everything what you need is here https://openwrt.org/submitting-patches  and also in Git documentation itself to know why there is Git commit description

Comment (stangri) [reply net/pbr/Makefile:35]: The outdated packages have not been part of the OpenWrt repository since 21.02, I think it's time to remove any references to them and not drag them on forever.
